docs/griptape-framework/structures/task-memory.md

```diff
@@ -1,89 +1,320 @@
 ## Overview
-Task Memory augments activity inputs and outputs with storage capabilities. It's a way to keep any data generated by tools off prompt but still allow LLMs to operate on that data remotely. This is useful in the following scenarios:
+
+Task Memory is a powerful feature of Griptape that allows you to control where the data returned by Tools is stored. This is useful in the following scenarios:
+
 
 * **Security requirements**: many organizations don't want data to leave their cloud for regulatory and security reasons.
-* **Long textual content**: when textual content returned by tools can't fit in the token limit, it's often useful to perform operations on it in a separate process, not in the main LLM.
-* **Non-textual content**: tools can generate images, videos, PDFs, and other non-textual content that can be stored in memory and acted upon later by other tools.
+* **Long textual content**: when textual content returned by Tools can't fit in the token limit, it's often useful to perform operations on it in a separate process, not in the main LLM.
```

Comment:
Should probably not say "process" since it's still the same system process. Can't think of a better word though right now.

Reply:
suggestions: operation, thread, or action?